src/runtime/windows_d3d12compute_arm.cpp

```diff
@@ -0,0 +1,13 @@
+// NOTE(marcos): this file is a trivial wrapper around the "core" d3d12 back-end
+// located in "d3d12compute.cpp"; this wrapper is used as a mean to "decorate" the
+// module such that the proper build flags and target triple can be determined based
+// on the prefix (windows_) and suffix (_arm) when building the runtime module.
+
+#ifndef BITS_64
+// Don't emit compiler messages (via #pragma message) here: some environments might
```

Comment:
So does this mean the windows 32-bit d3d runtime module is built but is empty?

Reply:
Yes, pretty much.
Do we have some sort of "null/parrot" runtime module to reroute instead?